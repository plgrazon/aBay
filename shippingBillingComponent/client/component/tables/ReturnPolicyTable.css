:local(.return) {
  border-collapse: collapse;
  font-size: 16px;
  font-family: helvetica, arial, sans-serif;
  width: 98%;
  text-align: left;
  margin-left: auto;
  margin-right: auto;
}

:local(.return-header) {
  border-bottom: 1px solid black;
  border-collapse: collapse;
  background-color: #e8eaed;
}

:local(.return-outside-border) {
  border: 1px solid black;
}

:local(.return-lower-border) {
  border-bottom: 1px solid black;
}

:local(.return-policy-header) {
  border-bottom: 1px solid black;
  border-collapse: collapse;
  background-color: #e8eaed;
}

:local(th) {
  height: 40px;
  text-align: left;
}

:local(td) {
  height: 30px;
  text-align: left;
}
