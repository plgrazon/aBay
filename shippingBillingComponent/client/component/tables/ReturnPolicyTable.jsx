import React from 'react';

import styles from './ReturnPolicyTable.css';

const ReturnPolicyTable = () => (
  <div className={styles['return-outside-border']}>
    <h3>Return Policy</h3>
    <table className={styles.return}>
      <thead>
        <tr className={styles['return-header']}>
          <th>After receiving item, contact seller within</th>
          <th>Refund will be given as</th>
          <th>Return shipping</th>
        </tr>
      </thead>
      <tbody>
        <tr className={styles['return-lower-border']}>
          <td>30 days</td>
          <td>Money back</td>
          <td>Seller pays for return shipping</td>
        </tr>
        <tr>
          <td colSpan="3">
            <p>
                Refer to <a href="https://www.ebay.com/help/buying/returns-refunds/return-item-refund?id=4041#section3">eBay Return policy</a> for more details. You are covered by the <a href="https://www.ebay.com/help/policies/ebay-money-back-guarantee-policy/ebay-money-back-guarantee-policy?id=4210">eBay Money Back Guarantee</a> if you receive an item that is not as described in the listing.
            </p>
          </td>
        </tr>
        <tr className={styles['return-policy-header']}>
          <th colSpan="3">Return policy details</th>
        </tr>
        <tr>
          <td colSpan="3">
            All buyers remorse returns or returns for refused delivery will be subject to 20% restocking fee.
            Any items that are returned not complete or with damages pieces will not be refunded.
          </td>
        </tr>
      </tbody>
    </table>
  </div>
);

export default ReturnPolicyTable;
