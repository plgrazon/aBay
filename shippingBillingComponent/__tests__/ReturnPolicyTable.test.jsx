import ReturnPolicyTable from '../client/component/tables/ReturnPolicyTable';

describe('Component: ReturnPolicyTable', () => {
  const wrapper = shallow(<ReturnPolicyTable />);

  it('ReturnPolicyTable should be defined', () => {
    expect(wrapper).toBeDefined();
  });

  it('ReturnPolicyTable should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('ReturnPolicyTable should render table', () => {
    expect(wrapper.find('table').length).toBe(1);
  });
});
