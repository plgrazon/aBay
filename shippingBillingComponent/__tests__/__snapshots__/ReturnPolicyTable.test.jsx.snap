// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: ReturnPolicyTable ReturnPolicyTable should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ReturnPolicyTable />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3>
          Return Policy
        </h3>,
        <table>
          <thead>
            <tr>
              <th>
                After receiving item, contact seller within
              </th>
              <th>
                Refund will be given as
              </th>
              <th>
                Return shipping
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>
                30 days
              </td>
              <td>
                Money back
              </td>
              <td>
                Seller pays for return shipping
              </td>
            </tr>
            <tr>
              <td
                colSpan="3"
              >
                <p>
                  Refer to 
                  <a
                    href="https://www.ebay.com/help/buying/returns-refunds/return-item-refund?id=4041#section3"
                  >
                    eBay Return policy
                  </a>
                   for more details. You are covered by the 
                  <a
                    href="https://www.ebay.com/help/policies/ebay-money-back-guarantee-policy/ebay-money-back-guarantee-policy?id=4210"
                  >
                    eBay Money Back Guarantee
                  </a>
                   if you receive an item that is not as described in the listing.
                </p>
              </td>
            </tr>
            <tr>
              <th
                colSpan="3"
              >
                Return policy details
              </th>
            </tr>
            <tr>
              <td
                colSpan="3"
              >
                All buyers remorse returns or returns for refused delivery will be subject to 20% restocking fee. Any items that are returned not complete or with damages pieces will not be refunded.
              </td>
            </tr>
          </tbody>
        </table>,
      ],
      "className": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Return Policy",
        },
        "ref": null,
        "rendered": "Return Policy",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <thead>
              <tr>
                <th>
                  After receiving item, contact seller within
                </th>
                <th>
                  Refund will be given as
                </th>
                <th>
                  Return shipping
                </th>
              </tr>
            </thead>,
            <tbody>
              <tr>
                <td>
                  30 days
                </td>
                <td>
                  Money back
                </td>
                <td>
                  Seller pays for return shipping
                </td>
              </tr>
              <tr>
                <td
                  colSpan="3"
                >
                  <p>
                    Refer to 
                    <a
                      href="https://www.ebay.com/help/buying/returns-refunds/return-item-refund?id=4041#section3"
                    >
                      eBay Return policy
                    </a>
                     for more details. You are covered by the 
                    <a
                      href="https://www.ebay.com/help/policies/ebay-money-back-guarantee-policy/ebay-money-back-guarantee-policy?id=4210"
                    >
                      eBay Money Back Guarantee
                    </a>
                     if you receive an item that is not as described in the listing.
                  </p>
                </td>
              </tr>
              <tr>
                <th
                  colSpan="3"
                >
                  Return policy details
                </th>
              </tr>
              <tr>
                <td
                  colSpan="3"
                >
                  All buyers remorse returns or returns for refused delivery will be subject to 20% restocking fee. Any items that are returned not complete or with damages pieces will not be refunded.
                </td>
              </tr>
            </tbody>,
          ],
          "className": undefined,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <tr>
                <th>
                  After receiving item, contact seller within
                </th>
                <th>
                  Refund will be given as
                </th>
                <th>
                  Return shipping
                </th>
              </tr>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <th>
                    After receiving item, contact seller within
                  </th>,
                  <th>
                    Refund will be given as
                  </th>,
                  <th>
                    Return shipping
                  </th>,
                ],
                "className": undefined,
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "After receiving item, contact seller within",
                  },
                  "ref": null,
                  "rendered": "After receiving item, contact seller within",
                  "type": "th",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Refund will be given as",
                  },
                  "ref": null,
                  "rendered": "Refund will be given as",
                  "type": "th",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Return shipping",
                  },
                  "ref": null,
                  "rendered": "Return shipping",
                  "type": "th",
                },
              ],
              "type": "tr",
            },
            "type": "thead",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <tr>
                  <td>
                    30 days
                  </td>
                  <td>
                    Money back
                  </td>
                  <td>
                    Seller pays for return shipping
                  </td>
                </tr>,
                <tr>
                  <td
                    colSpan="3"
                  >
                    <p>
                      Refer to 
                      <a
                        href="https://www.ebay.com/help/buying/returns-refunds/return-item-refund?id=4041#section3"
                      >
                        eBay Return policy
                      </a>
                       for more details. You are covered by the 
                      <a
                        href="https://www.ebay.com/help/policies/ebay-money-back-guarantee-policy/ebay-money-back-guarantee-policy?id=4210"
                      >
                        eBay Money Back Guarantee
                      </a>
                       if you receive an item that is not as described in the listing.
                    </p>
                  </td>
                </tr>,
                <tr>
                  <th
                    colSpan="3"
                  >
                    Return policy details
                  </th>
                </tr>,
                <tr>
                  <td
                    colSpan="3"
                  >
                    All buyers remorse returns or returns for refused delivery will be subject to 20% restocking fee. Any items that are returned not complete or with damages pieces will not be refunded.
                  </td>
                </tr>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <td>
                      30 days
                    </td>,
                    <td>
                      Money back
                    </td>,
                    <td>
                      Seller pays for return shipping
                    </td>,
                  ],
                  "className": undefined,
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "30 days",
                    },
                    "ref": null,
                    "rendered": "30 days",
                    "type": "td",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Money back",
                    },
                    "ref": null,
                    "rendered": "Money back",
                    "type": "td",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Seller pays for return shipping",
                    },
                    "ref": null,
                    "rendered": "Seller pays for return shipping",
                    "type": "td",
                  },
                ],
                "type": "tr",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <td
                    colSpan="3"
                  >
                    <p>
                      Refer to 
                      <a
                        href="https://www.ebay.com/help/buying/returns-refunds/return-item-refund?id=4041#section3"
                      >
                        eBay Return policy
                      </a>
                       for more details. You are covered by the 
                      <a
                        href="https://www.ebay.com/help/policies/ebay-money-back-guarantee-policy/ebay-money-back-guarantee-policy?id=4210"
                      >
                        eBay Money Back Guarantee
                      </a>
                       if you receive an item that is not as described in the listing.
                    </p>
                  </td>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <p>
                      Refer to 
                      <a
                        href="https://www.ebay.com/help/buying/returns-refunds/return-item-refund?id=4041#section3"
                      >
                        eBay Return policy
                      </a>
                       for more details. You are covered by the 
                      <a
                        href="https://www.ebay.com/help/policies/ebay-money-back-guarantee-policy/ebay-money-back-guarantee-policy?id=4210"
                      >
                        eBay Money Back Guarantee
                      </a>
                       if you receive an item that is not as described in the listing.
                    </p>,
                    "colSpan": "3",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        "Refer to ",
                        <a
                          href="https://www.ebay.com/help/buying/returns-refunds/return-item-refund?id=4041#section3"
                        >
                          eBay Return policy
                        </a>,
                        " for more details. You are covered by the ",
                        <a
                          href="https://www.ebay.com/help/policies/ebay-money-back-guarantee-policy/ebay-money-back-guarantee-policy?id=4210"
                        >
                          eBay Money Back Guarantee
                        </a>,
                        " if you receive an item that is not as described in the listing.",
                      ],
                    },
                    "ref": null,
                    "rendered": Array [
                      "Refer to ",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "eBay Return policy",
                          "href": "https://www.ebay.com/help/buying/returns-refunds/return-item-refund?id=4041#section3",
                        },
                        "ref": null,
                        "rendered": "eBay Return policy",
                        "type": "a",
                      },
                      " for more details. You are covered by the ",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "eBay Money Back Guarantee",
                          "href": "https://www.ebay.com/help/policies/ebay-money-back-guarantee-policy/ebay-money-back-guarantee-policy?id=4210",
                        },
                        "ref": null,
                        "rendered": "eBay Money Back Guarantee",
                        "type": "a",
                      },
                      " if you receive an item that is not as described in the listing.",
                    ],
                    "type": "p",
                  },
                  "type": "td",
                },
                "type": "tr",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <th
                    colSpan="3"
                  >
                    Return policy details
                  </th>,
                  "className": undefined,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Return policy details",
                    "colSpan": "3",
                  },
                  "ref": null,
                  "rendered": "Return policy details",
                  "type": "th",
                },
                "type": "tr",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <td
                    colSpan="3"
                  >
                    All buyers remorse returns or returns for refused delivery will be subject to 20% restocking fee. Any items that are returned not complete or with damages pieces will not be refunded.
                  </td>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "All buyers remorse returns or returns for refused delivery will be subject to 20% restocking fee. Any items that are returned not complete or with damages pieces will not be refunded.",
                    "colSpan": "3",
                  },
                  "ref": null,
                  "rendered": "All buyers remorse returns or returns for refused delivery will be subject to 20% restocking fee. Any items that are returned not complete or with damages pieces will not be refunded.",
                  "type": "td",
                },
                "type": "tr",
              },
            ],
            "type": "tbody",
          },
        ],
        "type": "table",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            Return Policy
          </h3>,
          <table>
            <thead>
              <tr>
                <th>
                  After receiving item, contact seller within
                </th>
                <th>
                  Refund will be given as
                </th>
                <th>
                  Return shipping
                </th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>
                  30 days
                </td>
                <td>
                  Money back
                </td>
                <td>
                  Seller pays for return shipping
                </td>
              </tr>
              <tr>
                <td
                  colSpan="3"
                >
                  <p>
                    Refer to 
                    <a
                      href="https://www.ebay.com/help/buying/returns-refunds/return-item-refund?id=4041#section3"
                    >
                      eBay Return policy
                    </a>
                     for more details. You are covered by the 
                    <a
                      href="https://www.ebay.com/help/policies/ebay-money-back-guarantee-policy/ebay-money-back-guarantee-policy?id=4210"
                    >
                      eBay Money Back Guarantee
                    </a>
                     if you receive an item that is not as described in the listing.
                  </p>
                </td>
              </tr>
              <tr>
                <th
                  colSpan="3"
                >
                  Return policy details
                </th>
              </tr>
              <tr>
                <td
                  colSpan="3"
                >
                  All buyers remorse returns or returns for refused delivery will be subject to 20% restocking fee. Any items that are returned not complete or with damages pieces will not be refunded.
                </td>
              </tr>
            </tbody>
          </table>,
        ],
        "className": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Return Policy",
          },
          "ref": null,
          "rendered": "Return Policy",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <thead>
                <tr>
                  <th>
                    After receiving item, contact seller within
                  </th>
                  <th>
                    Refund will be given as
                  </th>
                  <th>
                    Return shipping
                  </th>
                </tr>
              </thead>,
              <tbody>
                <tr>
                  <td>
                    30 days
                  </td>
                  <td>
                    Money back
                  </td>
                  <td>
                    Seller pays for return shipping
                  </td>
                </tr>
                <tr>
                  <td
                    colSpan="3"
                  >
                    <p>
                      Refer to 
                      <a
                        href="https://www.ebay.com/help/buying/returns-refunds/return-item-refund?id=4041#section3"
                      >
                        eBay Return policy
                      </a>
                       for more details. You are covered by the 
                      <a
                        href="https://www.ebay.com/help/policies/ebay-money-back-guarantee-policy/ebay-money-back-guarantee-policy?id=4210"
                      >
                        eBay Money Back Guarantee
                      </a>
                       if you receive an item that is not as described in the listing.
                    </p>
                  </td>
                </tr>
                <tr>
                  <th
                    colSpan="3"
                  >
                    Return policy details
                  </th>
                </tr>
                <tr>
                  <td
                    colSpan="3"
                  >
                    All buyers remorse returns or returns for refused delivery will be subject to 20% restocking fee. Any items that are returned not complete or with damages pieces will not be refunded.
                  </td>
                </tr>
              </tbody>,
            ],
            "className": undefined,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <tr>
                  <th>
                    After receiving item, contact seller within
                  </th>
                  <th>
                    Refund will be given as
                  </th>
                  <th>
                    Return shipping
                  </th>
                </tr>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <th>
                      After receiving item, contact seller within
                    </th>,
                    <th>
                      Refund will be given as
                    </th>,
                    <th>
                      Return shipping
                    </th>,
                  ],
                  "className": undefined,
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "After receiving item, contact seller within",
                    },
                    "ref": null,
                    "rendered": "After receiving item, contact seller within",
                    "type": "th",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Refund will be given as",
                    },
                    "ref": null,
                    "rendered": "Refund will be given as",
                    "type": "th",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Return shipping",
                    },
                    "ref": null,
                    "rendered": "Return shipping",
                    "type": "th",
                  },
                ],
                "type": "tr",
              },
              "type": "thead",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <tr>
                    <td>
                      30 days
                    </td>
                    <td>
                      Money back
                    </td>
                    <td>
                      Seller pays for return shipping
                    </td>
                  </tr>,
                  <tr>
                    <td
                      colSpan="3"
                    >
                      <p>
                        Refer to 
                        <a
                          href="https://www.ebay.com/help/buying/returns-refunds/return-item-refund?id=4041#section3"
                        >
                          eBay Return policy
                        </a>
                         for more details. You are covered by the 
                        <a
                          href="https://www.ebay.com/help/policies/ebay-money-back-guarantee-policy/ebay-money-back-guarantee-policy?id=4210"
                        >
                          eBay Money Back Guarantee
                        </a>
                         if you receive an item that is not as described in the listing.
                      </p>
                    </td>
                  </tr>,
                  <tr>
                    <th
                      colSpan="3"
                    >
                      Return policy details
                    </th>
                  </tr>,
                  <tr>
                    <td
                      colSpan="3"
                    >
                      All buyers remorse returns or returns for refused delivery will be subject to 20% restocking fee. Any items that are returned not complete or with damages pieces will not be refunded.
                    </td>
                  </tr>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <td>
                        30 days
                      </td>,
                      <td>
                        Money back
                      </td>,
                      <td>
                        Seller pays for return shipping
                      </td>,
                    ],
                    "className": undefined,
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "30 days",
                      },
                      "ref": null,
                      "rendered": "30 days",
                      "type": "td",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Money back",
                      },
                      "ref": null,
                      "rendered": "Money back",
                      "type": "td",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Seller pays for return shipping",
                      },
                      "ref": null,
                      "rendered": "Seller pays for return shipping",
                      "type": "td",
                    },
                  ],
                  "type": "tr",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <td
                      colSpan="3"
                    >
                      <p>
                        Refer to 
                        <a
                          href="https://www.ebay.com/help/buying/returns-refunds/return-item-refund?id=4041#section3"
                        >
                          eBay Return policy
                        </a>
                         for more details. You are covered by the 
                        <a
                          href="https://www.ebay.com/help/policies/ebay-money-back-guarantee-policy/ebay-money-back-guarantee-policy?id=4210"
                        >
                          eBay Money Back Guarantee
                        </a>
                         if you receive an item that is not as described in the listing.
                      </p>
                    </td>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <p>
                        Refer to 
                        <a
                          href="https://www.ebay.com/help/buying/returns-refunds/return-item-refund?id=4041#section3"
                        >
                          eBay Return policy
                        </a>
                         for more details. You are covered by the 
                        <a
                          href="https://www.ebay.com/help/policies/ebay-money-back-guarantee-policy/ebay-money-back-guarantee-policy?id=4210"
                        >
                          eBay Money Back Guarantee
                        </a>
                         if you receive an item that is not as described in the listing.
                      </p>,
                      "colSpan": "3",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          "Refer to ",
                          <a
                            href="https://www.ebay.com/help/buying/returns-refunds/return-item-refund?id=4041#section3"
                          >
                            eBay Return policy
                          </a>,
                          " for more details. You are covered by the ",
                          <a
                            href="https://www.ebay.com/help/policies/ebay-money-back-guarantee-policy/ebay-money-back-guarantee-policy?id=4210"
                          >
                            eBay Money Back Guarantee
                          </a>,
                          " if you receive an item that is not as described in the listing.",
                        ],
                      },
                      "ref": null,
                      "rendered": Array [
                        "Refer to ",
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": "eBay Return policy",
                            "href": "https://www.ebay.com/help/buying/returns-refunds/return-item-refund?id=4041#section3",
                          },
                          "ref": null,
                          "rendered": "eBay Return policy",
                          "type": "a",
                        },
                        " for more details. You are covered by the ",
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": "eBay Money Back Guarantee",
                            "href": "https://www.ebay.com/help/policies/ebay-money-back-guarantee-policy/ebay-money-back-guarantee-policy?id=4210",
                          },
                          "ref": null,
                          "rendered": "eBay Money Back Guarantee",
                          "type": "a",
                        },
                        " if you receive an item that is not as described in the listing.",
                      ],
                      "type": "p",
                    },
                    "type": "td",
                  },
                  "type": "tr",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <th
                      colSpan="3"
                    >
                      Return policy details
                    </th>,
                    "className": undefined,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Return policy details",
                      "colSpan": "3",
                    },
                    "ref": null,
                    "rendered": "Return policy details",
                    "type": "th",
                  },
                  "type": "tr",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <td
                      colSpan="3"
                    >
                      All buyers remorse returns or returns for refused delivery will be subject to 20% restocking fee. Any items that are returned not complete or with damages pieces will not be refunded.
                    </td>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "All buyers remorse returns or returns for refused delivery will be subject to 20% restocking fee. Any items that are returned not complete or with damages pieces will not be refunded.",
                      "colSpan": "3",
                    },
                    "ref": null,
                    "rendered": "All buyers remorse returns or returns for refused delivery will be subject to 20% restocking fee. Any items that are returned not complete or with damages pieces will not be refunded.",
                    "type": "td",
                  },
                  "type": "tr",
                },
              ],
              "type": "tbody",
            },
          ],
          "type": "table",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
